### DISCUSSION TITLE
Changes to default quest order

### DISCUSSION THREAD

**Discussion by 10992-osm** [streetcomplete/StreetComplete#3998] [Q&A]
How does changes in the app's default quest order affect users who have modified that order themselves on their own device?  Also, when new quests are added, where will the be added in the list, at a specified numbered position, after a particular quest, or before a particular quest?

**Comment by matkoniecz** [comment]
Not sure will you consider it helpful answer - but I have no idea and I would need to look at code and/or experiment.
